client/source/components/questionsAnswers/InputQuestion.jsx

```diff
@@ -1,21 +1,21 @@
 import React, { useContext } from 'react';
-import ProdContext from '../../ProdContext';
-
+// import ProdContext from '../../ProdContext';
+import { InputContainer } from './styles/Containers.styled';
 
 const InputQuestion = function CreateInputQuestionComponent() {
-  const { prodID, setProdID } = useContext(ProdContext);
+  // const { prodID, setProdID } = useContext(ProdContext);
 
   return (
-    <div>
-      Product ID from InputQuestion: {prodID}
+    <InputContainer>
+      {/* Product ID from InputQuestion: {prodID} */}
       <input
         type="text"
         placeholder="Have a Question? Search For Answers"
       // onChange={(e) => setQuestInput(e.target.value)}
       // value={questInput}
       />
-      <button type="button" onClick={() => setProdID('new product ID')}>Change ID</button>
-    </div>
+      <button type="button" onClick={() => setProdID('new product ID')}>Go!</button>
```

Comment:
some commented code broke this line

Reply:
Thank you for all of the great comments. Sorry about the comments, but they have all been taken care of now. Thank you for bringing it to my attention!